journey/android/4.0.0/getting_started.md

```diff
@@ -0,0 +1,411 @@
+| `allowedId` | `List<String>` | ✗ | If you want to use only a small subset of the public transport objects in the Journey search (List of navitia uris) | `listOf("commercial_mode:Bus", "line:1")` |
+| `firstSectionModes` | `Set<String>` | ✗ | List of modes to use at the begining of the journey | `setOf("walking", "bike", "car", "bss", "ridesharing")` |
+| `lastSectionModes` | `Set<String>` | ✗ | List of modes to use at the end of the journey | `setOf("walking", "bike", "car", "bss", "ridesharing")` |
+| `count` | `Int` | ✗ | The number of journeys that will be displayed | `3` |
+| `minJourneys` | `Int` | ✗ | The minimum number of journeys that will be displayed | `3` |
+| `maxJourneys` | `Int` | ✗ | The maximum number of journeys that will be displayed | `10` |
+| `addPoiInfos` | `Set<String>` | ✗ | Allow the display of the availability in real time for bike share and car park | `setOf("bss_stands", "car_park")` |
+| `directPathMode` | `Set<String>` | ✗ | Allow full journeys with section modes | `setOf("car", "bike")` |
+| `physicalModes` | `Set<String>` | ✗ | Allowed physical modes in the coverage | `setOf("physical_mode:Metro", "physical_mode:Bus")` |
+| `dataFreshness` | `DataFreshness` | ✗ | To indicate if theoretical or realtime data are requested | `DataFreshness.REALTIME` |
+| `travelerType` | `TravelerType` | ✗ | To give extra information about the traveler state | `TravelerType.TRAVELER_TYPE_STANDARD` |
+
+</div>
+
+### Lines
+
+The `lines` is a JSON array of a line configuration.
+
+<div markdown="1">
+
+| Value | Type | Required | Description |
+| --- | --- |:---:| --- |
+| `code` | String | ✓ | Line code identifier |
+| `icon_res` | String | ✓ | String resource ID of the line icon |
+| `mode_id` | String | ✓ | Navitia commercial name of the line |
+
+</div>
+
+### Modes
+
+The `modes` is a JSON array of a transport mode configuration.
+
+<div markdown="1">
+
+| Value | Type | Required | Description |
+| --- | --- |:---:| --- |
+| `id` | String | ✓ | Navitia commercial name of the transport mode |
+| `icon_res` | String | ✓ | String resource ID of the mode icon |
+
+</div>
+
+- Using JSON file
+
+The JSON file should be added to the `assets` folder of your project.
+Please check the example below to know more about the structure of the configuration JSON file
+
+```json
+{
+  "lines": [
+    {
+      "code": "a",
+      "icon_res": "ic_metro_a",
+      "commercial": "Metro"
+    },
+    {
+      "code": "C1",
+      "icon_res": "ic_bus_c1",
+      "commercial": "Bus"
+    },
+    {
+      "code": "C2",
+      "icon_res": "ic_bus_c2",
+      "commercial": "Bus"
+    }
+  ],
+  "modes": [
+    {
+      "commercial": "Metro",
+      "icon_res": "ic_metro"
+    },
+    {
+      "commercial": "Bus",
+      "icon_res": "ic_bus"
+    },
+    {
+      "commercial": "Coach",
+      "icon_res": "ic_bus"
+    }
```

Comment:
I think we have to remove this. There's no coach line above. There is a risk this bring confusion to the reader.